index.js

```diff
@@ -1,6 +1,6 @@
 'use strict';
-var fs = require('fs');
-var path = require('path');
+const fs = require('fs');
+const path = require('path');
```

Comment:
this may have broken support for older versions of node

Reply:
totally did.